packages/flutter/lib/src/cupertino/nav_bar.dart

```diff
@@ -339,6 +347,19 @@ class CupertinoNavigationBar extends StatefulWidget implements ObstructingPrefer
   /// {@endtemplate}
   final Color? backgroundColor;
```

Comment:
Can you add something explaining how this will interact with `automaticBackGroundVisibility`? The sample code may need to be updated too.